File: code/community/1304606576094482442/opt-out-model-improvement-dotnet-sdk.go
// Go example using Deepgram SDK
// Ensure 'github.com/deepgram/deepgram-go-sdk' is installed
// For environment variables, use a package like 'godotenv'

package main

import (
    "fmt"
    "os"

    "github.com/deepgram/deepgram-go-sdk/deepgram"
)

func main() {
    apiKey := os.Getenv("DEEPGRAM_API_KEY")
    audioUrl := os.Getenv("AUDIO_FILE_URL")

    dg := deepgram.New(apiKey)

    options := deepgram.PreRecordedOptions{
        URL: audioUrl,
        MipOptOut: true,
    }

    response, err := dg.TranscribePr(

File: code/community/1304606576094482442/opt-out-model-improvement-dotnet-sdk.cs
// C# example using Deepgram SDK
// Ensure 'DeepgramApi' NuGet package is installed
// For environment variables, use a .env file or IDE settings

using System;
using System.Threading.Tasks;
using Deepgram;

namespace DeepgramExample
{
    class Program
    {
        static async Task Main(string[] args)
        {
            var apiKey = Environment.GetEnvironmentVariable("DEEPGRAM_API_KEY");
            var audioUrl = Environment.GetEnvironmentVariable("AUDIO_FILE_URL");

            var deepgramClient = new DeepgramClient(apiKey);

            var options = new Deepgram.Options.PreRecordedTranscriptionOptions {
                Url = audioUrl,
                MipOptOut = true
            };

            var response = await deepgramClient.Transcription.PrerecordedAsync(options);

            Console.WriteLine(response);
        }
    }
}